src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -42,7 +42,7 @@ public FunctionsController(WebScriptHostManager scriptHostManager, WebHookReceiv
 
             // Authorize the request
             SecretManager secretManager = (SecretManager)controllerContext.Configuration.DependencyResolver.GetService(typeof(SecretManager));
-            HttpBindingMetadata httpFunctionMetadata = (HttpBindingMetadata)function.Metadata.InputBindings.FirstOrDefault(p => p.Type == BindingType.HttpTrigger);
+            HttpTriggerBindingMetadata httpFunctionMetadata = (HttpTriggerBindingMetadata)function.Metadata.InputBindings.FirstOrDefault(p => p.Type == BindingType.HttpTrigger);
```

Comment:
I see. I still get confused about this. Maybe a side question, but is it fair to say that you can never have an http output binding without an http trigger input binding?